samples/snippets/update_rollout_default.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Game Servers sample for updating the rollout of a game
+server deployment to set the default config.
+
+Example usage:
+    python update_rollout_default.py --project-id <project-id> --deployment-id <deployment-id> --config-id <config-id>
+"""
+
+import argparse
+
+from google.cloud import gaming
+from google.cloud.gaming_v1.types import game_server_deployments
+from google.protobuf import field_mask_pb2 as field_mask  # type: ignore
+
+
+# [START gaming_update_rollout_default]
```

Comment:
For here, and all tags, please use the same tags as from the Go samples, so everything is consistent.  eg https://github.com/GoogleCloudPlatform/golang-samples/blob/master/gaming/servers/update_rollout_default.go#L17

Reply:
Done. Thanks